crates/pallet-domains/src/staking.rs

```diff
@@ -153,19 +141,21 @@ pub(crate) fn do_nominate_operator<T: Config>(
     let operator_pool = OperatorPools::<T>::get(operator_id).ok_or(Error::UnknownOperator)?;
 
     ensure!(!operator_pool.is_frozen, Error::OperatorPoolFrozen);
+
+    let updated_total_deposit = match PendingDeposits::<T>::get(operator_id, nominator_id.clone()) {
+        None => amount,
+        Some(existing_deposit) => existing_deposit
+            .checked_add(&amount)
+            .ok_or(Error::BalanceOverflow)?,
+    };
+
     ensure!(
-        amount >= operator_pool.minimum_nominator_stake,
+        updated_total_deposit >= operator_pool.minimum_nominator_stake,
```

Comment:
If we can check `(existing_deposit + updated_total_deposit) >= operator_pool.minimum_nominator_stake` then https://github.com/subspace/subspace/pull/1629#discussion_r1257297303 can be resolved.

Reply:
That might not be correct. Instead this check is for a new nominator and skips for existing nominator